### PR TITLE
Feature/replace tag pulldown select filter with table column filter

### LINKED ISSUE #294: Replace Tag Pulldown Select Filter With Table Column Filter
On the Tags page, the pulldown select filter should be removed. Instead, The tag listing table should allow the user to filter by tag.

image

### PR DESCRIPTION
**Pull request by YohannParis**
Fix #294 

- Added a filter button on the `Category` header column
- Change the `CustomOptionsFilter` component to be more straightforward
- General tidying up

### CODE REVIEW

File: packages/webapp/src/components/lists/TagsList/index.tsx [hunk 1]
```diff
@@ -18,20 +17,17 @@ import { AddTagForm } from '~forms/AddTagForm'
...
 import { useMobileColumns, usePageColumns } from './columns'
-const logger = createLogger('tagsList')
 
 interface TagsListProps {
 	title?: string
 }
 
 export const TagsList: StandardFC<TagsListProps> = wrap(function TagsList({ title }) {
-	const { t, c } = useTranslation(Namespace.Tags)
+	const { t /*, c*/ } = useTranslation(Namespace.Tags)
```
Comment (karlolsonuc) [inline packages/webapp/src/components/lists/TagsList/index.tsx:30]: Should this comment just be removed now?

File: packages/webapp/src/constants/TAG_CATEGORIES.ts [hunk 2]
```diff
@@ -9,5 +9,4 @@ export const TAG_CATEGORIES: TagCategory[] = [
 	TagCategory.Program,
 	TagCategory.Grant,
 	TagCategory.Other
-	// TagCategory.All
```
Comment (karlolsonuc) [inline packages/webapp/src/constants/TAG_CATEGORIES.ts:12]: Again, wonder if we could cut this.

**Review by karlolsonuc — APPROVED** [review]
Left a couple note about comments we could cut. Otherwise seems to be good, and it works on my local.

File: packages/webapp/src/components/lists/TagsList/index.tsx (hunk 1 above)
Comment (YohannParis) [reply packages/webapp/src/components/lists/TagsList/index.tsx:30]: I did in my other PR

File: packages/webapp/src/constants/TAG_CATEGORIES.ts (hunk 2 above)
Comment (YohannParis) [reply packages/webapp/src/constants/TAG_CATEGORIES.ts:12]: I did in my other PR

**Pull request merged**